Tests/FluentAssertions.Specs/Types/MethodInfoSelectorSpecs.cs

```diff
@@ -367,6 +394,14 @@ internal class TestClassForMethodSelectorWithNonInheritableAttributeDerived : Te
         public override void PublicVirtualVoidMethodWithAttribute() { }
     }
 
+    internal class TestClassForMethodSelectorWithAsync
+    {
+        public async virtual void PublicVirtualVoidAsyncMethod()
+        {
+            await Task.FromResult(42);
+        }
```

Comment:
Like in `ClassWithAllMethodsAsync`, this could be written as 
```cs
public async Task PublicAsyncDoNothing()
{
    await Task.Yield();
}
```

E.g. the `virtual` is irrelevant for this test.

Reply:
Thanks. I was looking for a way to have minimal code for an async method. :-)